feature/settings/src/main/res/values/strings.xml

```diff
@@ -83,5 +83,24 @@
     <string name="stabilization_description_on">Stabilization On</string>
     <string name="stabilization_description_high_quality">Stabilization High Quality</string>
     <string name="stabilization_description_off">Stabilization Off</string>
-    <string name="stabilization_description_unsupported">Stabilization unsupported</string>
+    <string name="stabilization_description_unsupported_device">Stabilization unsupported by device</string>
+    <string name="stabilization_description_unsupported_fps">Stabilization unsupported due to frame rate</string>
+
+
+    <!-- Target Fps setting strings -->
+    <string name="fps_title">Set Frame Rate</string>
+
+    <string name="fps_description_unavailable">No fixed frame rates supported</string>
+    <string name="fps_description_none">Auto Frame Rate</string>
+    <string name="fps_description_15">15 fps </string>
+    <string name="fps_description_30">30 fps</string>
+    <string name="fps_description_60">60 fps</string>
```

Comment:
You should only need a single format string for all of these:
`<string name="fps_description">%d fps</string>`

You can then get get the string with: `stringResources(R.string.fps_description, integerFps)`